recipe/meta.yaml

```diff
@@ -11,20 +11,20 @@ source:
   folder: src/
 
 build:
-  number: 0
-  skip: true  # [py<38]
+  number: 1
+  noarch: python
```

Comment:
We could add a comment to explain the logic. If I'm not mistaken it's to help with adding support for new Python versions to conda right?

Reply:
correct, we're suffering from a circular dependency issue, see GH Actions link above

Reply:
```suggestion
  # build as noarch to avoid circular dependency issues when adding support for new Python versions
  noarch: python
```

Reply:
Ok, thanks. Can you add a comment in the recipe so that we don't change it back by accident?

Reply:
Yup! see https://github.com/AnacondaRecipes/conda-libmamba-solver-feedstock/pull/21/commits/e050a18a5d221419c9161837b6447ec9bacc4beb